Assets/MXR.SDK/Runtime/Types/DeviceTypes.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace MXR.SDK {
+    public class DeviceData {
+        public bool isQfbDevice;
```

Comment:
For clarity of use, can we add some other fields here as well?

firmwareVersion, model, and manufacturer perhaps?

Reply:
Done! Also added the storage fields

Reply:
Thank you!

Reply:
Wait, not done - maybe waiting on a push?

Reply:
Added to the top of the stack @johnmanagexr 

Keeping one commit per PR as this stack is short anyway

Reply:
Got it - with graphite, you can rewrite history (and graphite preserves the different versions for PR review). It actually force pushed by default if you use their commands. I can show you this quickly after standup if you'd like!

It's normally easier to insert these type of edits in the same branch, so you can address code review (and still keep it to one commit). For this stack, it doesn't matter.

Reply:
I'd like that! Let's parking lot post standup